### PR TITLE
chore: adding msgServer struct to ics27 controller

### PR DESCRIPTION
**Pull request by damiannolan**
## Description

- Adding `msgServer` struct to ics27 controller submodule

closes: #2229 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by chatton — APPROVED** [review]
LGTM nice job 👍

**Review by charleenfei — APPROVED** [review]
thanks!!

File: modules/apps/27-interchain-accounts/controller/keeper/msg_server.go [hunk 1]
```diff
@@ -12,10 +12,20 @@ import (
...
+
+// NewMsgServerImpl returns an implementation of the ICS27 MsgServer interface
+// for the provided Keeper.
+func NewMsgServerImpl(keeper *Keeper) types.MsgServer {
+	return &msgServer{Keeper: keeper}
+}
 
 // RegisterAccount defines a rpc handler for MsgRegisterAccount
-func (k Keeper) RegisterAccount(goCtx context.Context, msg *types.MsgRegisterAccount) (*types.MsgRegisterAccountResponse, error) {
+func (k msgServer) RegisterAccount(goCtx context.Context, msg *types.MsgRegisterAccount) (*types.MsgRegisterAccountResponse, error) {
```
Comment (crodriguezvega) [inline modules/apps/27-interchain-accounts/controller/keeper/msg_server.go:28]: Nit: should `k` be `ms` instead?

**Review by crodriguezvega — COMMENTED** [review]
Sorry, I see that this already merged, but I left just a nit...

**Pull request merged**